### PR TITLE
SIDE-1511 remove keyboard service.stream scan code

### PR DESCRIPTION
**Pull request by RusGraf**
Removing a scan code service from c#, node and GO implementations. As it is a dead code.

### CODE REVIEW

File: ldk/go/self-test-loop/main.go [hunk 1]
```diff
@@ -149,12 +149,7 @@ func (loop *Loop) listenToKeyboard() error {
 		return err
 	}
 
-	err = loop.sidekick.Keyboard().ListenScancode(loop.ctx, loop.keyboardScancodeHandler())
-	if err != nil {
-		loop.logger.Error("keyboard scancode listen error", err)
-	}
-
-	return err
+return err
```
Comment (DanielWagersOlive) [inline ldk/go/self-test-loop/main.go:152]: nit: missing indent

Comment (RusGraf) [reply ldk/go/self-test-loop/main.go]: Updated

File: ldk/node/package-lock.json [hunk 2]
```diff
@@ -1,8 +1,10679 @@
 {
   "name": "@oliveai/ldk",
-  "version": "2.0.0-alpha",
-  "lockfileVersion": 1,
+  "version": "2.0.0-beta",
```
Comment (DanielWagersOlive) [inline ldk/node/package-lock.json:3]: What happened here?

**Review by DanielWagersOlive — APPROVED** [review]

File: ldk/node/package-lock.json (hunk 2 above)
Comment (RusGraf) [reply ldk/node/package-lock.json:3]: Not sure actually, looks like it decided to move a line down.

**Pull request closed without merging**